cmd/oc-bundle/create.go

```diff
@@ -7,7 +7,15 @@ import (
 	"github.com/RedHatGov/bundle/pkg/bundle/create"
 )
 
+type createOpts struct {
+	segSize   int64
```

Comment:
Thoughts on hiding this for now?

Reply:
How about putting it in the imageset-config?